default.nix

```diff
@@ -7,9 +7,9 @@ let
   eval = cfg: lib.evalModules {
     modules = lib.singleton {
       # _file = toString input;
-      imports = lib.singleton { devices = cfg; };
+      imports = lib.singleton { disko.devices = cfg.disko.devices; };
```

Comment:
There could be a warning here to help user upgrading there configuration i.e. if no `disko.devices` is set in the file.

Reply:
what would the warning say? right now we cat a warning, that disko.devices is not set correcty, which should be kinda clear enough?